llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -6202,7 +6202,7 @@ bool AArch64TargetLowering::useSVEForFixedLengthVectorVT(
 
   // All SVE implementations support NEON sized vectors.
```

Comment:
I'm not sure we want to go overboard with giving the impression SME is distinct from SVE because the nuances are going to confuse people, with this a prime example.  Whilst SVE does truly support NEON size vectors, the same is not true for SME where the support is simulated though the use of predication.  In general when we mean "SVE instructions" I'd rather we just say SVE and then better document the SVE instructions that are not provided by SME, which are significant fewer in number.

Reply:
I have tried to clarify the comment without making it sound like SME is distinct from SVE.